src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Statistics/FootnoteControllerTests.cs

```diff
@@ -171,49 +169,46 @@ public FootnoteControllerTests()
         }
 
         [Fact]
-        public async Task Post_CreateFootnote_Returns_Ok()
+        public async Task CreateFootnote()
         {
-            var result = await _controller.CreateFootnote(ReleaseId, new FootnoteCreateViewModel()
-            {
-                Content = "Sample footnote",
-                Filters = new List<Guid>(),
-                FilterGroups = new List<Guid>(),
-                FilterItems = new List<Guid>(),
-                Indicators = new List<Guid>(),
-                Subjects = new List<Guid>()
-            });
-
-            Assert.IsType<FootnoteViewModel>(result.Value);
+            var result = await _controller.CreateFootnote(ReleaseId,
+                new FootnoteCreateRequest
+                {
+                    Content = "Sample footnote",
+                    Filters = SetOf<Guid>(),
+                    FilterGroups = SetOf<Guid>(),
+                    FilterItems = SetOf<Guid>(),
+                    Indicators = SetOf<Guid>(),
+                    Subjects = SetOf<Guid>()
+                });
+
+            result.AssertOkResult();
         }
 
         [Fact]
-        public async Task Get_Footnotes_Returns_Ok()
+        public async Task GetFootnotes()
         {
             var result = await _controller.GetFootnotes(ReleaseId);
-            Assert.IsAssignableFrom<IEnumerable<FootnoteViewModel>>(result.Value);
-        }
 
-        [Fact]
-        public async Task Put_UpdateFootnote_Returns_Ok()
-        {
-            var result = await _controller.UpdateFootnote(ReleaseId, FootnoteId, new FootnoteUpdateViewModel
-            {
-                Content = "Updated sample footnote",
-                Filters = new List<Guid>(),
-                FilterGroups = new List<Guid>(),
-                FilterItems = new List<Guid>(),
-                Indicators = new List<Guid>(),
-                Subjects = new List<Guid>()
-            });
-
-            Assert.IsType<FootnoteViewModel>(result.Value);
+            result.AssertOkResult();
         }
 
         [Fact]
-        public async Task Delete_DeleteFootnote_Returns_Ok()
+        public async Task UpdateFootnote()
```

Comment:
Is there a reason we've lost our DeleteFootnote() tests?  Not that they were testing much particularly :)

Reply:
I was thinking about dropping the whole of the controller tests because generally the controller tests we've written in the past don't test very much.

In the case of this controller though, it has a complicated methods which build view models. So I've kept all test methods that end up returning view models which pass through code like `BuildFootnoteViewModel`, but removed this test which wasn't really testing anything.

Reply:
Yeah fair enough!